shotover-proxy/tests/redis_int_tests/basic_driver_tests.rs

```diff
@@ -21,18 +21,30 @@ const STRESS_TEST_MULTIPLIER: usize = 1;
 #[cfg(not(debug_assertions))]
 const STRESS_TEST_MULTIPLIER: usize = 100;
 
+async fn assert_nil(cmd: &mut Cmd, connection: &mut Connection) {
```

Comment:
Can these helpers live in the mod.rs file for this module?

Reply:
sure, that sounds reasonable

Reply:
Ended up putting it in its own submodule, that tends to be the preferred way in rust.

Reply:
LGTM. If there's other existing helpers in for redis that could live there I'm fine with including that in this PR if you want to move them. Otherwise lets move forward with this.

Reply:
nah, theres nothing else in there I would consider a helper